### DISCUSSION TITLE
Have troube changing the keyboard shortcuts of labelme.exe

### DISCUSSION THREAD

**Discussion by Esperer105** [wkentaro/labelme#1049] [Q&A / Help] [answered]
Hello, I try to change the keyboard shortcuts by editing the [labelme/labelme/config/default_config.yaml](https://github.com/wkentaro/labelme/blob/f8097a4b19c79ca3328022cfbc351f2c7bff20d5/labelme/config/default_config.yaml#L71-L106) as your answer in #988. 
Then I follow the instructions to standalone executable file. But the new generated labelme.exe file doesn't have any changes on the key board shortcuts.
 
![image](https://user-images.githubusercontent.com/77342125/172516723-46c4c71e-c6ac-4b5d-a9d7-f60175badc4a.png)
So is it the right way to change the shortcuts of the labelme.exe? if not, how to change the keyboard shortcuts?
Thanks a lot!

**Comment by wkentaro** [comment, marked as answer]
You can edit `~/.labelmerc` instead of `default_config.yml`. If you change `default_config.yml`, you need to regenerate the standalone executable to have the effect (and also `rm ~/.labelmerc`).

**Reply by Esperer105** [reply]
Thanks a lot! I have changed my shortcuts according the readme
